app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @dish.name %></h1>
+<h3><%= @dish.description %></h3>
+<p>Ingredients: <% @dish.ingredients.each do |ingredient| %>
+<%= ingredient.name %>
+<% end %>
+</p>
+<p>Total Calories: <%= @dish.total_calories %></p>
+<p>Chef <%= @dish.chef.name %></p>
+
+<section id="add_ingredients">
+<%= form_with url: "/dishes/#{@dish.id}/dish_ingredients", method: :post, data: { turbo: false } do |f| %>
+<%= f.label :ingredient_id %>
+<%= f.text_field :ingredient_id %>
+<%= f.submit "Submit" %>
+<% end %>
+</section>
```

Comment:
Good use of `<section></section>` here.

Your form is nearly perfect. The only change that needs to be made is the url that the form is posting to. It should be posting to `"/dishes/#{@dish.id}.ingredients"`, other than that, this is perfect. Great job!